.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
     strategy:
       matrix:
         include:
-        - kind-node-version: v1.18.8
```

Comment:
I would suggest to have the minimal version, that supports the leader election, tested as well.

Reply:
We'd need an e2e test that actually verifies leader election, because this can only determined at runtime. Currently we don't have that so a 1.16 cluster is "useless"